docs/hazmat/primitives/asymmetric/rsa.rst

```diff
@@ -485,6 +518,43 @@ Key interfaces
             instance.
 
 
+.. class:: RSAPrivateKeyWithSerialization
+
+    .. versionadded:: 0.8
+
+    Extends :class:`RSAPrivateKey`.
+
+    .. method:: private_numbers()
+
+        Create a
+        :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers`
+        object.
+
+        :returns: An
+            :class:`~cryptography.hazmat.primitives.asymmetric.rsa.RSAPrivateNumbers`
+            instance.
+
+    .. method:: as_bytes(encoding, format, encryption_algorithm)
```

Comment:
Why `private_numbers` but not `private_bytes`? This seems inconsistent.

Would `private_as_numbers` / `private_as_bytes` make sense?